app/build.gradle

```diff
@@ -64,6 +64,7 @@ dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     implementation 'junit:junit:4.12'
     implementation 'com.android.support:appcompat-v7:28.0.0'
+
```

Comment:
If there are no relevant changes in a file, there is no point adding it to commit.